recipes/pytide/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+cxx_compiler_version:  # [osx]
+  - 8                  # [osx]
```

Comment:
Is this needed? Pinning ahead of conda-forge's global pins can make it harder to install packages for users.